tracing/src/lib.rs

```diff
@@ -192,8 +192,8 @@
 //!
 //! ## Using the Macros
 //!
-//! The [`span!`] and [`event!`] macros use fairly similar syntax, with some
-//! exceptions.
+//! The [`span!`] and [`event!`] macros as well as the `#[tracing::instrument]` attribute
```

Comment:
Can we make this link to the `#[instrument]` attribute's docs as well?

Also, I think that we probably just want to use the un-prefixed name --- we don't say `tracing::span!` or `tracing::event!` here:
```suggestion
//! The [`span!`] and [`event!`] macros as well as the `#[tracing::instrument]` attribute
```
```suggestion
//! The [`span!`] and [`event!`] macros as well as the `#[instrument]` attribute
```